plugins/packages/airtable/lib/index.ts

```diff
@@ -9,11 +9,14 @@ export default class AirtableQueryService implements QueryService {
 
   async run(sourceOptions: SourceOptions, queryOptions: QueryOptions): Promise<QueryResult> {
     let result = {};
+    let apiToken = '';
     let response = null;
     const operation = queryOptions.operation;
     const baseId = queryOptions.base_id;
     const tableName = queryOptions.table_name;
-    const apiKey = sourceOptions.api_key;
+
+    if (sourceOptions.api_key) apiToken = sourceOptions.api_key;
```

Comment:
please add a comment that this is for backward compatibility and needs migration to be removed later on